COMPONENT_GUIDE.md

```diff
@@ -1,74 +1,18 @@
-- [Globally available components](#globally-available-components)
-  - [Video](#video)
-    - [Usage](#usage)
-  - [Intro](#intro)
-    - [Usage](#usage-1)
-  - [Steps](#steps)
-    - [Usage](#usage-2)
-  - [Step](#step)
-    - [Usage](#usage-3)
-- [A code example](#a-code-example)
-- [Another code example](#another-code-example)
-  - [Code blocks](#code-blocks)
-    - [Usage](#usage-4)
-  - [Callouts](#callouts)
-    - [Usage](#usage-5)
-  - [Related Resources](#related-resources)
-    - [Maximum references](#maximum-references)
-    - [Resource population logic](#resource-population-logic)
-    - [Order of priority](#order-of-priority)
-    - [Resource site tags](#resource-site-tags)
-  - [Announcement Banner](#announcement-banner)
-
-# Globally available components
-
-## Video
-
-`<Video />` component provides formatting for videos in markdown.
+# Component Documentation
 
-### Usage
+## Global Components
 
-The Video component requires two props:
+Refer to the [New Relic Gatsby Theme](https://github.com/newrelic/gatsby-theme-newrelic/blob/develop/packages/gatsby-theme-newrelic/README.md)
+component documentation for details on all the components used on this site.
 
-- `id`: the video ID
-- `type`: the host of the video. Accepted values are:
-  - `wistia`
-  - `youtube`
+## Developer Site Components
 
-```md
-<Video id="zxunt1u1as" type="wistia"/>
-```
+Below are the components that are unique to the Developer Site. These components support patterns that are specific to the Developer Site and are not used by other sites.
 
 ## Intro
 
 The `<Intro />` component provides formatting for the title and introduction of the markdown document.
 
-### Usage
-
-It takes the title provided in the Frontmatter and accepts plain text for the description. An example of Frontmatter that will have a title of **Example Guide**:
-
-```
----
-path: '/example'
-duration: '30 min'
-title: 'Example Guide'
-template: 'GuideTemplate'
-description: 'Example guide page'
----
-```
-
-It also accepts a `<Video />` component as a child, which it will place on the left side of the description.
-
-```md
-<Intro>
-This is a description for the markdown guide.
-
-<Video id="zxunt1u1as" type="youtube"/>
-</Intro>
-```
-
-If there is a more than plain text and a `<Video />` (such as a code snippet or another component) the content will be posted on the left side below the description.
```

Comment:
It looks like this stuff is still relevant to the `Intro` component. Would you mind restoring this section? The rest of the changes look great!

Reply:
it is done